.circleci/config.yml

```diff
@@ -54,6 +54,44 @@ jobs:
           name: Print urls to log
           command: |
             cat broken_external_urls.txt
+  
+  netlify_manual:
+    docker:
+      - image: cimg/node:14.21.3-browsers
+    resource_class: xlarge
+    steps:
+      - run:
+          name: Has secrets?
+          command: |
+            if [ -z "$NETLIFY_AUTH_TOKEN" ] || [ -z "$NETLIFY_SITE_ID" ]; then
+              echo "Missing required secrets."
+              exit 1
+            fi
+      # Step to cancel workflow
+      - run:
+          name: Cancel workflow if failure
```

Comment:
Is this step required? If any step fails it will cancel the whole job. I'm not sure how this will work and also how to actually get into this step.

Reply:
Its a precautionary measure to ensure that the necessary secrets (NETLIFY_AUTH_TOKEN and NETLIFY_SITE_ID) are provided before proceeding with the job.

While it's true that any step failure will cancel the entire job, having this explicit check ensures that the job fails early if essential secrets are missing, providing a more informative error message than if the job fails due to a step relying on those secrets.

The step is not strictly required for the job to function, but it adds an extra layer of safety and clarity to our configuration. We can remove the step also.

For now, I'll remove this step as the build will itself fail if the secrets are not there.

Reply:
Yes, this isn't required. Removed it.